raimed2Front/src/views/HomeView.vue

```diff
@@ -14,11 +32,10 @@ import AuthenticatedPageLayout from '@/layouts/AuthenticatedViewLayout.vue';
         aboutir au diagnostic dans une situation clinique donnée.<br />
       </p>
       <p class="w-full text-center">A vous de jouer !</p>
-      <h1 class="title mt-6">Comment fonctionne le site ?</h1>
-      <p class="text">
-        Sur votre gauche, vous pouvez retrouvez un menu en cliquant sur l’icône. Vous pouvez ensuite
-        choisir un patient et faire un diagnostic.
-      </p>
+      <div class="flex flex-col m-4 space-y-4" v-if="role == Role.TEACHER">
+        <ActionButton on-click="" label="Visualisation des patients / Création de nouveaux patients" :color="Color.Orange"></ActionButton>
```

Comment:
```suggestion
        <ActionButton :onClick="() => console.log('Not implemented yet')" label="Visualisation des patients / Création de nouveaux patients" :color="Color.Orange"></ActionButton>
```
The property on-click isn't the same as the custom property onClick we've created for this component.
You should use the correct property `onClick`.